great3sims/psf.py

```diff
@@ -1382,10 +1382,22 @@ class DrawPSFBuilder(PSFBuilder):
     def __init__(self, obs_type, multiepoch, shear_type, draw_psf_src):
         self.obs_type = obs_type
         self.draw_psf_src = os.path.abspath(draw_psf_src)
+        filename = os.path.basename(self.draw_psf_src)
```

Comment:
I think you're substantially changing the semantics of the ``DrawPSFBuilder`` class here. Sadly the original doesn't seem to be overly blessed with documentation (at least, not in this file), however, for starters, it looks as though it used to assume ``draw_psf_src`` was a FITS file, and now you have it as a plain text catalogue.

This is not wrong as such, but it makes me wonder whether this is the best approach. Are you certain you will never need to use the original ``DrawPSFBuilder`` interface? Might your changes confuse others who have to work with this code in future? Did you consider subclassing rather than simply replacing the original?